docs/index.qmd

```diff
@@ -40,6 +41,40 @@ about:
 ::: {#about}
 :::
 
+
+## Try it out!
+
+
+```{python}
+#| echo: false
+#| output: asis
+from urllib.parse import urlencode
+
+lines = """
+%pip install numpy pandas tzdata
+import pyodide_js
+await pyodide_js.loadPackage("https://storage.googleapis.com/ibis-wasm-wheels/pyarrow-16.0.0.dev2661%2Bg9bddb87fd-cp311-cp311-emscripten_3_1_46_wasm32.whl")
+del pyodide_js
+%clear
+%pip install 'ibis-framework[duckdb]'
+from ibis.interactive import *
+penguins = ibis.read_csv("penguins.csv")
+penguins.head()
+"""
+
+params = [
+    ("toolbar", "1"),
+    ("theme", "JupyterLab Night"),
```

Comment:
It'd be nice to have the theme switch with the quarto toggle. I'm not sure how to do that. I think the procedure would be something like:

1. find the button element
2. attach an onclick listener to that button that sets the theme

Reply:
this is also an issue for general plotting, there was some prior art there but it looked semi-complicated when I checked